internal/planner/planner.go

```diff
@@ -2363,6 +2363,10 @@ func rewrittenVar(vars map[ast.Var]ast.Var, k ast.Var) ast.Var {
 	return rw
 }
 
+func dont() ([][]*ast.Rule, []ir.Operand, int, bool) {
+	return nil, nil, 0, false
+}
```

Comment:
unrelated cleanup: it seemed silly to allocate that function over and over when it's really just a shortcut.